tools/img/sixpack/contig.fasta

```diff
@@ -0,0 +1,184 @@
+>NODE_133
```

Comment:
Could you add a more descriptive name please?

Reply:
I've added the SRA run name as a prefix!